source/drivers/java-drivers.txt

```diff
@@ -0,0 +1,41 @@
+.. _java-language-center:
+
+====================
+Java MongoDB Drivers
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Introduction
+------------
+
+:doc:`Java Driver </drivers/java>` is the recommended MongoDB Java Driver.
+
+:doc:`Reactive Streams Driver </drivers/reactive-streams>` is the
+recommended MongoDB Java Driver for asynchronous stream processing and
+reactive streams interoperability.
```

Comment:
I wonder if it would work better to lead with "For asynchronous stream processing ..., Reactive Streams Driver is the recommended..."

Your call.

Reply:
Works for me. I was just mimicking the Python driver landing page (https://docs.mongodb.com/ecosystem/drivers/python/), but your suggestion provides better positioning of the subject and object.